website/views/shared/modals/members.jade

```diff
@@ -114,8 +114,9 @@ script(type='text/ng-template', id='modals/abuse-flag.html')
     button.btn.btn-danger(ng-click='reportAbuse(user, abuseObject, groupId)')=env.t("abuseFlagModalButton")
 
 script(type='text/ng-template', id='modals/remove-member.html')
-  .modal-header
+  .modal-header(class='text-center')
     h4=env.t('sureKick')
+    p(class='modal-subheading') {{removeMemberData.member.profile.name}}
```

Comment:
this should be `p.modal-subheading {{::removeMemberData.member.profile.name}}` (notice the `::` to signify one-time-binding`